src/game/AI/ScriptDevAI/scripts/outland/tempest_keep/botanica/boss_laj.cpp

```diff
@@ -134,14 +160,36 @@ struct boss_lajAI : public CombatAI
         DoBroadcastText(EMOTE_SUMMON, m_creature);
 
         if (m_creature->GetVictim())
-            m_creature->GetMotionMaster()->MoveChase(m_creature->GetVictim());
+        {
+            SetCombatMovement(true);
+            SetCombatScriptStatus(false);
+
+            m_creature->MeleeAttackStart(m_creature->GetVictim());
+            m_creature->SetTarget(m_creature->GetVictim());
+        }
     }
 };
 
+struct AllergicReaction : public AuraScript
+{
+    bool OnCheckProc(Aura* /*aura*/, ProcExecutionData& data) const override
+    {
+        // Player with aura 34697 can proc Aura 34700 on other Players
```

Comment:
What about pets?

Reply:
I only can use current cata classic to test this, but on cata:
Pet can get hit by spell 34697 (if pet is tanking)
Pet can't get hit by spell 34700 (if player tanks and has aura 34697)

But i actually see that this change will prevent spreading to allys too.

Reply:
Yes, there is a simple function IsPlayerControlled that will do what you want :)

Reply:
I removed this change for now. Using IsPlayerControlled didnt help either. Will need to debug this more.